clang/docs/ReleaseNotes.rst

```diff
@@ -803,6 +803,8 @@ Bug Fixes to C++ Support
 - Fixed an incorrect lambda scope of generic lambdas that caused Clang to crash when computing potential lambda
   captures at the end of a full expression. (#GH115931)
 - Clang no longer rejects deleting a pointer of incomplete enumeration type. (#GH99278)
+- Fixed recognition of ``std::initializer_list`` when it's surrounded with ``extern "C++"`` and exported
```

Comment:
I guess it is worth mentioning the pattern comes from MSVC's std module implementation.